### PR TITLE
Update rn force flag

### PR DESCRIPTION
**Pull request by dansterenson**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/37772

## depends on
https://github.com/demisto/demisto-sdk/pull/1368

## Description
Added the ability to force update release notes with the --force flag or -f flag. After speaking to @ShahafBenYakir, we agreed that we only need the ability to automatically bump a version with the update release notes command and then manually filling in the changes.

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Review by roysagi — APPROVED** [review]

**Review by reutshal — APPROVED** [review]
Nice 🔥

**Comment by ShahafBenYakir** [comment]
@dansterenson Please update docs as well

**Comment by dansterenson** [comment]
@roysagi can you please review my new addition to the demisto_sdk/commands/update_release_notes/README.md ?

**Comment by ShahafBenYakir** [comment]
Also here https://xsoar.pan.dev/docs/concepts/demisto-sdk#update-release-notes

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Changelog
 * Refactor **update-release-notes** command - new classs manager, split some logic to smaller functions, added docsrings and type annotations.
-
+* Added a  *--force* flag to the **update release notes** command, so the update can takes place even without a change.
```
Comment (roysagi) [inline CHANGELOG.md:3]:
```suggestion
* Added a  *--force* flag to the **update-release-notes** command, so the update can take place even without a change in the pack.
```

File: demisto_sdk/__main__.py [hunk 2]
```diff
@@ -1044,6 +1044,9 @@ def merge_id_sets(**kwargs):
 @click.option(
     '--all', help="Update all changed packs", is_flag=True
 )
+@click.option(
+    '-f', '--force', help="Force update rn (even if not required).", is_flag=True
```
Comment (roysagi) [inline demisto_sdk/__main__.py:1048]:
```suggestion
    '-f', '--force', help="Force update release notes for a pack (even if not required).", is_flag=True
```

**Pull request merged**